cmd/daemon.go

```diff
@@ -39,7 +40,7 @@ const (
 // override the defaultStateURL to use https scheme
 var defaultStateURL = url.URL{
 	Scheme: "https",
-	Host:   net.JoinHostPort(dcos.DNSRecordLeader, strconv.Itoa(dcos.PortMesosMaster)),
+	Host:   net.JoinHostPort(dcosGo.DNSRecordLeader, strconv.Itoa(dcosGo.PortMesosMaster)),
```

Comment:
it looks like some constants are pulled out of `github.com/dcos/dcos-go/dcos` (e.g. `RoleMaster`) into the `dcos` package in this project - should `DNSRecordLeader` and `PortMesosMaster` be as well to avoid importing both of these `dcos` pkgs here?

Reply:
Good point. I moved the whole definition of `defaultStateUrl` to dcos package.